_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and exists an [instrument][] to disable the [keyboard event][]. A disabled event implies that the event is disabled when the `getModifierState` method returns `false`, therefore meeting the **remap** expectation.
+
+```html
+<html>
+  <head>
+    <title>Passed Example 2</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({id: 'singleShortcut', shortcutKey: '+'}); activateShortcuts();">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <label>
+      <input type="checkbox" onclick="toggleDisabled('singleShortcut', !this.checked)" checked>
+      Toggle single character keyboard shortcut
+    </label>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 3
+
+This [HTML document][] has two [keyboard events][keyboard event] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and for each [keyboard event][] exists an [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`.
+
+```html
+<html>
+  <head>
+    <title>Passed Example 3</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({id: 'firstShortcut', shortcutKey: '+'}); registerShortcut({id: 'secondShortcut', shortcutKey: 'a'}); activateShortcuts();">
+    <label for="target">Add to list (press "+" or "a" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      <div>Remap shortcut</div>
+      <div>
+        <label>
+          <input id="remap1" type="checkbox" onclick="toggleModifier('firstShortcut', this.checked)">
+          Use "ctrl" key together with the "+" key
+        </label>
+        <label>
+          <input id="remap2" type="checkbox" onclick="toggleModifier('secondShortcut', this.checked)">
+          Use "ctrl" key together with the "a" key
+        </label>
+      </div>
+    </div>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 4
+
+This [HTML document][] has two [keyboard events][keyboard event] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and exists an [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`. In this example, the same [instrument][] is used to **remap** both [keyboard events][keyboard event]
```

Comment:
```suggestion
This [HTML document][] is listening to [keyboard events][keyboard event] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and there exists an [instrument][] to **remap** those [keyboard events][keyboard event] that cause [change in content][] so that [same key][] events are [blocked][blocked event] unless `getModifierState("Control")` returns `true`. In this example, the same [instrument][] is used to **remap** all [keyboard events][keyboard event].
```